logpolicy/logpolicy.go

```diff
@@ -500,6 +501,9 @@ func New(collection string) *Policy {
 		},
 		HTTPC: &http.Client{Transport: newLogtailTransport(logtail.DefaultHost)},
 	}
+	if collection == "tailnode.log.tailscale.io" {
```

Comment:
optional: I wonder if this should be a constant

Reply:
Will do.